lib/performance_platform/gateway/unique_users.rb

```diff
@@ -0,0 +1,25 @@
+class PerformancePlatform::Gateway::UniqueUsers
+  def initialize(period:)
+    @period = period
+  end
+
+  def fetch_stats
+    {
+      count: result[:count].to_i,
+      metric_name: 'unique-users',
+      period: period.to_s
```

Comment:
What's the `to_s` about?

Reply:
Joys of dynamically typed language. Allows you to pass in symbols if you want. Though could be done in an initializer.

Reply:
Fair enough!